massgov/mayflower
# React/DP-12448: InputRadioGroup.

## Description
This adds support for Input and InputProvider to InputRadioGroup.

## Related Issue / Ticket

- [JIRA issue](https://jira.mass.gov/browse/DP-12448)

## Steps to Test


1. Pull the branch, go to Molecules -> InputRadioGroup and ensure that nothing has changed functionality wise for the component.
